components/Chips/src/MDCChipView.m

```diff
@@ -276,6 +276,14 @@ - (void)setEnableRippleBehavior:(BOOL)enableRippleBehavior {
   }
 }
 
+- (BOOL)allowsSelection {
+  return !self.enableRippleBehavior || self.rippleView.allowsSelection;
```

Comment:
would this mean that if ripple isn't enabled then allowSelection will always return YES, is this intended?

Reply:
Yes. My assumption is that this is the desired/current behavior of ink in chips, but I 'm actually not sure. Is it not the current behavior in ink?

Reply:
I believe this is only the case when using an MDCChipCollectionCell, whereas an MDCChipView goes from highlighted then back to normal. You will have to check and confirm though on the types of chips and in the Dragons examples. If there are chips that dont go into selected after tapping, then this should facilitate that here too.

Reply:
Because this is a Ripple API, when enableRippleBehavior is not enabled, the behavior is unpredictable.